sources/packages/backend/apps/api/src/health.controller.ts

```diff
@@ -1,10 +1,14 @@
 import { Controller, Get } from "@nestjs/common";
 import { AppService } from "./app.service";
 import { Public } from "./auth/decorators/public.decorator";
-@Controller()
-export class AppController {
+@Controller("health")
```

Comment:
Thanks for renaming the health endpoint. We need to ensure the deployment config would also reflect the new name.
https://github.com/bcgov/SIMS/blob/6f6118bd89f17076c0fb2d85f8d5472421142e4d/devops/openshift/api-deploy.yml#L221-L233

If possible, excluding the `api` prefix for the `health` route would also block its public access.
If can be done changing the below.
https://github.com/bcgov/SIMS/blob/675738c7e238983dd65dc115960fbd17fbd9b8eb/sources/packages/backend/apps/api/src/main.ts#L30-L31
to 
```ts
app.setGlobalPrefix("api", { exclude: ["health"] });
```

Reply:
Nice catch. I had the deploy-yml in my mind, but forgot while raising the PR. thanks for noticing it.